ingestr/src/frankfurter/__init__.py

```diff
@@ -13,25 +13,26 @@
     max_table_nesting=0,
 )
 def frankfurter_source(
-    table: str,
-    start_date: Optional[TAnyDateTime] = None,
-    end_date: Optional[TAnyDateTime] = None,
+    start_date: TAnyDateTime,
+    end_date: TAnyDateTime,
 ) -> Any:
     """
     A dlt source for the frankfurter.dev API. It groups several resources (in this case frankfurter.dev API endpoints) containing
     various types of data: currencies, latest rates, historical rates.
-
-    Returns the appropriate resource based on the provided parameters.
     """
-    # Determine which resource to return based on the `table` parameter
-    if table == "currencies":
-        return currencies()
-
-    elif table == "latest":
-        return latest()
+    dateTime = dlt.sources.incremental(

```

Comment:
minor nitpick: python conventions prefer `snake_case` over `camelCase`. The only exception to this rule is `CapitalCase`, usually reserved for classes